### ISSUE TITLE
[BUG] ng2-color-picker refuses to load

### ISSUE THREAD

**Issue by Wats0ns** [AndyMeps/ng2-color-picker#7]
Hello,
I just installed the plugin after following the README instructions. However, Angular fails to load due to this error coming from ng2-color-picker:
`XHR error (404 Not Found) loading http://localhost:5555/node_modules/ng2-color-picker.js`
This is the content of my app.module.ts:
```
import { ColorPickerModule } from 'ng2-color-picker';

@NgModule({
  imports: [ColorPickerModule, BrowserModule, HttpModule, AppRoutingModule, AboutModule, HomeModule, SharedModule.forRoot()],
  declarations: [AppComponent],
  providers: [{
    provide: APP_BASE_HREF,
    useValue: '<%= APP_BASE %>'
  }],
  bootstrap: [AppComponent]

})
```
I'm using angular2-seed.
What am I doing wrong ?
Thanks ;-)

**Comment by AndyMeps** [comment]
Hey WatsOns,

This line:

`XHR error (404 Not Found) loading http://localhost:5555/node_modules/ng2-color-picker.js`

Implies it's your implementation that is the issue and not the library. If you can provide me more details of your implementation I can look to help you.

**Comment by Wats0ns** [comment]
Well when you do a `npm install`, there is no such thing as `node_modules/ng2-color-picker.js`

**Comment by AndyMeps** [comment]
Exactly, this doesn't seem right, I've just created a new angular2-seed project, and had a minor issue with the package, I was getting:

`[default] C:\git\@andymeps\angular2-seed\src\app\app.module.ts:18:34 
Cannot find module 'ng2-color-picker'.`

Instead I had to use:

`import { ColorPickerModule} from 'ng2-color-picker/dist';`

Also it's in the develop branch readme.md but not the master branch - you need bootstrap css in index.html:

`<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">`

See if this works

Andy

**Comment by Wats0ns** [comment]
Unfortunately this does'nt fix it, I'm now getting this error:
`XHR error (404 Not Found) loading http://localhost:5555/node_modules/ng2-color-picker/dist.js`
Quentin
